images/macos/macos-15-Readme.md

```diff
@@ -32,12 +33,12 @@
 - Carthage 0.40.0
 - CocoaPods 1.16.2
 - Composer 2.8.8
-- Homebrew 4.4.30
+- Homebrew 4.4.31
 - NPM 10.9.2
 - Pip3 25.0.1 (python 3.13)
 - Pipx 1.7.1
 - RubyGems 3.6.8
-- Vcpkg 2025 (build from commit d5182f703b)
+- Vcpkg 2025 (build from commit 0d5cae1530)
```

Comment:
@erik-bershel @ijunaidm1 Why do we use random commits instead of aligning with the official releases?